boards/mkrzero/board_config.h

```diff
@@ -0,0 +1,15 @@
+#ifndef BOARD_CONFIG_H
+#define BOARD_CONFIG_H
+
+#define VENDOR_NAME "Arduino LLC"
+#define PRODUCT_NAME "Arduino MKRZero"
+#define VOLUME_LABEL "MKRZEROBOOT"
+
+#define BOARD_ID "SAMD21G18A-MKRZero-v0"
+
+//#define USB_VID 0x2341
```

Comment:
hmm what VID/PID is used if its not #defined'? we can assign one from the adafruit pool

Reply:
Both the Zero and MKR1000 had their VID/PID commented out, so I figured do the same to the MKRZero, as it's another Arduino board.

Without UF2 (e.g. running Blinky Arduino code):
VID: 2341
PID: 804F

With UF2 installed (no CircuitPython):
VID: 03EB
PID: 2402

With CircuitPython installed:
VID: 239A
PID: 8015

It looks like it's pulling from a default Atmel/Microchip (0x03EB VID) pool when it's just UF2 and it's getting 0x8015 from the Adafruit (0x239A) pool. Note that I'm loading Feather m0 Basic CircuitPython on it to test, so it might think it's a Feather m0.

Reply:
ooh ok yeah - alright hold on lets use these from Adafruit

> MKRZero           VID 0x239A    PID 0x0035  # UF2 Bootloader
>                                 PID 0x8035  # CircuitPython

@ShawnHymel please use the VID/PIDs above for UF2 and then CircuitPython, and put a note "Adafruit VID/PID assigned to MKRZero for hobby use"